packages/stark-ui/src/modules/date-picker/components/date-picker.component.html

```diff
@@ -0,0 +1,14 @@
+<mat-form-field>
+	<input matInput 
+		[disabled]="isDisabled"
+		[matDatepicker]="picker" 
+		[placeholder]="label | translate" 
+		[matDatepickerFilter]="dateFilterFnWrapper"
+		[min]="minDate"
+		[max]="maxDate"
+		[name]="pickerName"
+		[value]="date"
+		(dateChange)="onDateChange($event)">
+	<mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
```

Comment:
Same here about the id

Reply:
I think there is no way to set an id to this toggle element